components/AudioPlayer.tsx

```diff
@@ -0,0 +1,388 @@
+          }
+        }}
+        onClick={() => {
+          if (isMobile()) {
+            setActiveVolume(!activeVolume);
+          }
+        }}
+      >
+        {Boolean(volume) ? (
+          <img
+            src="/images/volume-icon.svg"
+            alt="play"
+            width={22}
+            height={16}
+            style={
+              activeVolume
+                ? {
+                    filter:
+                      "invert(47%) sepia(66%) saturate(2546%) hue-rotate(356deg) brightness(101%) contrast(99%)",
+                  }
+                : {}
+            }
+          />
+        ) : (
+          <img
+            src="/images/mute-volume-icon.svg"
+            alt="play"
+            width={22}
+            height={16}
+            style={
+              activeVolume
+                ? {
+                    filter:
+                      "invert(47%) sepia(66%) saturate(2546%) hue-rotate(356deg) brightness(101%) contrast(99%)",
+                  }
+                : {}
+            }
+          />
+        )}
+      </IconButton>
+      <Box
+        component="span"
+        sx={(theme) => ({
+          opacity: 0,
+          pointerEvents: "none",
+          position: "absolute",
+          display: "flex",
+          width: "157px",
+          height: "38px",
+          padding: "13px",
+          transform: "rotate(-90deg)",
+          bottom: "120px",
+          right: "-60px",
+          backgroundColor:
+            theme.palette.mode === "light" ? "#f2f5f7" : "#33393f",
+          borderRadius: "4px",
+          justifyContent: "center",
+          alignItems: "center",
+          transition: "opacity 500ms ease-in-out",
+          "&[closing]": {
+            opacity: 0,
+          },
+          "&[open]": {
+            opacity: 1,
+            pointerEvents: "auto",
+          },
+          "& input": {
+            height: "4px",
+            width: "100%",
+            "--seek-before-width": 0,
+            backgroundColor:
+              theme.palette.mode === "light" ? "white" : "#18191d",
+            appearance: "none",
+            borderRadius: "4px",
+            position: "relative",
+            outline: "none",
+            cursor: "pointer",
+            "&::before": {
+              content: '""',
+              height: "4px",
+              width: "var(--seek-before-width)",
+              backgroundColor:
+                theme.palette.mode === "light" ? "#000000" : "#f2f5f7",
+              borderRadius: "4px",
+              position: "absolute",
+              zIndex: 2,
+            },
+            "&::-webkit-slider-thumb": {
+              opacity: 0,
+            },
+          },
+        })}
+        onMouseEnter={() => {
+          if (!isMobile()) clearTimeout(timeoutRef.current);
+        }}
+        onMouseLeave={() => {
+          if (!isMobile()) setActiveVolume(false);
+        }}
+      >
+        <input
+          type="range"
+          defaultValue="100"
```

Comment:
we don't modify it
?

Reply:
Remove default value here. It is useless here.